test/fixtures/config/self_service_user.json

```diff
@@ -0,0 +1,22 @@
+{
+  "description": "defines agreed fixture config that can be relied upon by browser testing tools requiring defined pact/contract data",
+  "config": {
+    "users": [
+      { "isPrimary": "true",
```

Comment:
what does isPrimary mean?

Reply:
This is just means the primary user as far as default contract tests are based around. If we wanted to add other users to test other operations, we could use other filters to include/exclude this user. It doesn't relate to anything other than the fixture config (i.e. has no meaning/purpose outside of it).

Was a first crack at how to do a fixed config, so happy to change it.